keyboards/handwired/dactyl_lightcycle/readme.md

```diff
@@ -0,0 +1,38 @@
+# Dactyl Lightcycle
+
+![TopElev](https://pbs.twimg.com/media/C_lE5BUU0AEufDT?format=jpg)
```

Comment:
this should be rehosted on imgur or the like.

Reply:
amended as requested